catalogs/catalog-lakehouse-paimon/src/main/java/org/apache/gravitino/catalog/lakehouse/paimon/PaimonCatalogPropertiesMetadata.java

```diff
@@ -41,9 +41,9 @@
 public class PaimonCatalogPropertiesMetadata extends BaseCatalogPropertiesMetadata {
 
   @VisibleForTesting public static final String GRAVITINO_CATALOG_BACKEND = "catalog-backend";
-  @VisibleForTesting public static final String PAIMON_METASTORE = "metastore";
-  @VisibleForTesting public static final String WAREHOUSE = "warehouse";
-  @VisibleForTesting public static final String URI = "uri";
+  public static final String PAIMON_METASTORE = "metastore";
+  public static final String WAREHOUSE = "warehouse";
+  public static final String URI = "uri";
```

Comment:
My comment from previous PR, can we make these variables be package public, I don't think they need to be public if they're only for tests.

Reply:
My understanding is that they are used outside of tests.

Reply:
> My comment from previous PR, can we make these variables be package public, I don't think they need to be public if they're only for tests.

@jerryshao 

I respect your opinion, but it was already used outside. What are your opinions on this?

Reply:
I see, I'm fine with this.

Reply:
@jerryshao Thank you for your constructive comments.

Reply:
@justinmclean Did you have a good day today? If you have time, please check.